<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SlugsPadCameraControl</class>
 <widget class="QWidget" name="SlugsPadCameraControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>238</width>
    <height>317</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 170, 0);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>1</number>
   </property>
   <property name="spacing">
    <number>1</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(135, 206, 235);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>213</width>
            <height>156</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="lbPixel">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>----</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lbDirection">
            <property name="text">
             <string>----</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="btZoomIn">
            <property name="toolTip">
             <string extracomment="Zoom In"/>
            </property>
            <property name="text">
             <string>Zoom In</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btHome">
            <property name="toolTip">
             <string extracomment="Move Home"/>
            </property>
            <property name="text">
             <string>Home</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btZoomOut">
            <property name="toolTip">
             <string extracomment="Zoom Out"/>
            </property>
            <property name="text">
             <string>Zoom Out</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <zorder>frame</zorder>
  <zorder></zorder>
  <zorder></zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
